docker/Dockerfile

```diff
@@ -182,6 +182,9 @@ COPY --from=build_image /project/avalon/common/python/dist/*.whl dist/
 COPY --from=build_image /project/avalon/common/crypto_utils/dist/*.whl dist/
 COPY --from=build_image /project/avalon/sdk/dist/*.whl dist/
 
+# Copy Scripts
+COPY ./scripts /project/avalon/scripts
```

Comment:
We need scripts since apps run in avalon-shell container and thiis script invokes generic client application. There will be more scripts added in this directory. So for now copying the entire scripts folder in avalon-shell container. Later if we decide to create separate containers for applications we can revisit this.

Reply:
Why can't we generic client inside the shell container? As we are running in docker.

Reply:
I think copying scripts is not needed. This kubernetes setup is to let people try, setup and run whatever they want. Anyone can go to shell container and run whatever they want.

Reply:
@pankajgoyal2 The scripts are copied since kubernetes job pod uses it to start the job(generic_client) when pods are launched. Please suggest how to run kubernetes job pod without copying scripts to avalon shell container.